src/hugr/rewrite/outline_cfg.rs

```diff
@@ -128,26 +129,21 @@ impl Rewrite for OutlineCfg {
             let cfg = new_block_bldr
                 .cfg_builder(wires_in, outputs, extension_delta)
                 .unwrap();
-            let cfg_outputs = cfg.finish_sub_container().unwrap().outputs();
+            let cfg = cfg.finish_sub_container().unwrap();
             let predicate = new_block_bldr
                 .add_constant(ops::Const::simple_unary_predicate(), ExtensionSet::new())
                 .unwrap();
             let pred_wire = new_block_bldr.load_const(&predicate).unwrap();
-            new_block_bldr.set_outputs(pred_wire, cfg_outputs).unwrap();
-            h.insert_hugr(outer_cfg, new_block_bldr.hugr().clone())
-                .unwrap()
+            new_block_bldr
+                .set_outputs(pred_wire, cfg.outputs())
+                .unwrap();
+            let (new_block, node_map) = h
+                .insert_hugr(outer_cfg, new_block_bldr.hugr().clone())
+                .unwrap();
+            (new_block, *node_map.get(&cfg.node()).unwrap())
```

Comment:
The second element here should just be `new_block`, no?

Reply:
In which case all of the changes to return nodemaps are redundant and all we needed was to bind the result of `insert_hugr`

Reply:
Ah I see that's what was happening before and the Cfg is meant to be its child